endpoints/create_object.py

```diff
@@ -0,0 +1,14 @@
+import requests
+from endpoints.base_endpoint import Endpoint
+
+
+class CreateObject(Endpoint):
+    POST_CREATE_ENTITY = "/create"
+
+    def create_entity(self, payload):
+        self.response = requests.post(f"{self.URL}{self.POST_CREATE_ENTITY}", json=payload)
+        self.response_txt = self.response.text
```

Comment:
Необходимы создавать методы, в которых реализовано взаимодействие с эндпоинтом через конкретный метод, а в виде результата возвращался его ответ (response).
 ```suggestion
    def create_entity(self, payload):
        return requests.post(f"{self.URL}{self.POST_CREATE_ENTITY}", json=payload)
```